Production/src/BaseTupleProducer.cc

```diff
@@ -719,11 +764,19 @@ void BaseTupleProducer::FillEventTuple(const analysis::SelectionResultsBase& sel
             eventTuple().jets_csv.push_back(jet->bDiscriminator("pfCombinedInclusiveSecondaryVertexV2BJetTags"));
             eventTuple().jets_deepCsv_b.push_back(jet->bDiscriminator("pfDeepCSVJetTags:probb")); //new
             eventTuple().jets_deepCsv_bb.push_back(jet->bDiscriminator("pfDeepCSVJetTags:probbb")); //new
+            eventTuple().jets_deepCsv_c.push_back(jet->bDiscriminator("pfDeepCSVJetTags:probc")); //new
 //            eventTuple().jets_deepCsv_b_vs_all.push_back(jet->bDiscriminator("pfDeepCSVDiscriminatorsJetTags:BvsAll")); //sum of b and bb
             eventTuple().jets_rawf.push_back((jet->correctedJet("Uncorrected").pt() ) / p4.Pt());
             eventTuple().jets_mva.push_back(jet->userFloat("pileupJetId:fullDiscriminant"));
             eventTuple().jets_partonFlavour.push_back(jet->partonFlavour());
             eventTuple().jets_hadronFlavour.push_back(jet->hadronFlavour());
+            //jet resolution
+            JME::JetParameters parameters;
+            parameters.setJetPt(jet.GetMomentum().pt());
+            parameters.setJetEta(jet.GetMomentum().eta());
+            parameters.setRho(*rho);
+            float jet_resolution = resolution.getResolution(parameters);
+            eventTuple().jets_resolution.push_back(jet_resolution); // percentage
```

Comment:
- rho is important variable for jets, better to store it in eventTuple;
- is there some instructions on how to get resolution for fat jets?